detect.go

```diff
@@ -31,13 +31,10 @@ type PyProjectParser interface {
 // defined in the pyproject.toml under [tool.poetry.scripts]
 func Detect(pyProjectParser PyProjectParser) packit.DetectFunc {
```

Comment:
I think this is missing the logic to pass detection if the `BP_POETRY_RUN_TARGET` environment variable is set.

Reply:
You are correct, this was an oversight.